x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/chart_switch/chart_switch.scss

```diff
@@ -5,6 +5,19 @@
   }
 }
 
+.lnsChartSwitch__options {
+  width: 384px;
+
+  .lnsChartSwitch__option {
+    padding: $euiSizeS 0;
+  }
+
+  .euiSelectableList__groupLabel {
+    padding: $euiSizeS;
+  }
+
```

Comment:
Sorry, I did a bad job explaining my previous padding simplification comment. Let's get rid of these altogether. We'll address the padding in other changes.

```suggestion
```